com.civicrm.hrjobroles/CRM/Hrjobroles/BAO/HrJobRoles.php

```diff
@@ -47,6 +47,20 @@ public static function getContactRoles($cuid)  {
   }
 
   /**
+   * Check Contact if exist   .
+   *
+   * @param String $searchValue
+   * @param String $searchField
+   * @return Integer ( Contact ID or 0 if not exist)
+   */
+  public static function checkContact($searchValue, $searchField) {
```

Comment:
Why did you move this method up? I'm just curious
Also, since you touched it, what about renaming the method clearly describe what it really does? I think that something like checkIfContactExists of just contactExists (then you can have something like `if(contactExists())`) would be better than this generic "checkContact" that can means anything